web/satellite/src/components/common/VSearch.vue

```diff
@@ -5,7 +5,7 @@
     <input
         ref="input"
         v-model="searchQuery"
-        class="common-search-input"
+        :class="`${styleType}-search-input`"
```

Comment:
At this point it feels like a separate component makes more sense.

Or better yet, implement it such that it doesn't require the special case. I think the inputWidth change via code doesn't look right.